packages/@webex/media-helpers/src/webrtc-core.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable @typescript-eslint/no-misused-new */
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
+import {
+  AudioDeviceConstraints,
+  createCameraTrack as wcmeCreateCameraTrack,
```

Comment:
why does it have to be named as wcmeCreateCameraTrack

there is no change in the functionality

Reply:
wcmeCreateCameraTrack is a generic function, it takes a track class as a parameter so that it can instantiate the right class. SDK is calling it with the SDK's LocalCameraTrack class passed as that parameter